raiden/transfer/mediated_transfer/target.py

```diff
@@ -201,7 +231,10 @@ def handle_block(target_state, channel_state, block_number):
         events = [failed]
 
     elif target_state.state != 'waiting_close':  # only emit the close event once
+        # TODO: to be removed
         events = events_for_close(target_state, channel_state, block_number)
+
+        events = events_for_onchain_secretregister(target_state, channel_state, block_number)
```

Comment:
this is overwriting the previous value for events, either remove the previous line or use `extend`